src/app/rpmostree-builtin-kargs.cxx

```diff
@@ -266,6 +270,8 @@ rpmostree_builtin_kargs (int            argc,
   g_variant_dict_insert (&dict, "reboot", "b", opt_reboot);
   g_variant_dict_insert (&dict, "initiating-command-line", "s", invocation->command_line);
   g_variant_dict_insert (&dict, "lock-finalization", "b", opt_lock_finalization);
+  g_variant_dict_insert (&dict, "append-if-missing", "s", opt_append_if_missing_strings);
+  g_variant_dict_insert (&dict, "detele-if-present", "s", opt_delete_if_present_strings);
```

Comment:
This is not quite right. These are arrays, so it should be `^as` instead. Compare with the example I pointed to in https://github.com/coreos/rpm-ostree/pull/2713#discussion_r604407999 (specifically `vardict_insert_strv`).